src/panels/scene-viewer/module.ts

```diff
@@ -46,5 +46,9 @@ export const plugin = new PanelPlugin<PanelOptions>(ScenePanel)
         settings: {
           options: getVariableOptions(),
         },
+      })
+      .addTextInput({
```

Comment:
is this going to input the template variable name? it will be easier to addd `options: getVariableOptions(),` like what we did above

Reply:
Is that what that did? Ok I'll add that I thought that was something else